db/migrate/20230524122135_extend_policy_chain_my_sql.rb

```diff
@@ -0,0 +1,11 @@
+class ExtendPolicyChainMySql < ActiveRecord::Migration[5.2]
+  # Oracle and PostgreSQL do not need this as their `policies_config` has unlimited size
+
+  def up
+    change_column :proxies, :policies_config, :mediumtext if System::Database.mysql?
+  end
+
+  def down
+    change_column :proxies, :policies_config, :text if System::Database.mysql?
+  end
```

Comment:
I prefer to allow the option to rollback if something goes wrong, e.g. during deployment. Why would we want to prevent rollbacking?

Reply:
Because if operation fails, it remains the same. If operation once succeeds, we don't want it to ever try to rollback. And if db is broken, restore from backup will be needed. We don't change structure so we should never risk trying to reduce column width and risk a longer outage.

Reply:
The whole point is that if migration ever happens somehow, we better not provide any options to reverse because that would be a bigger risk - an additional DDL operation + possibility that a long record has already been added so reversal to fail. If a DBA wants so much, they can always issue a manual command, although I don't see any scenario where we would want to revert.